PYME/Analysis/points/DeClump/deClump.c

```diff
@@ -902,6 +902,135 @@ static PyObject * aggregateMin(PyObject *self, PyObject *args, PyObject *keywds)
     return NULL;*/
 }
 
+/*
+Aggregate data into clumps by taking the maximum. This assumes data has been sorted by clumpIndex
```

Comment:
This can also be achieved with a negative sign and aggregateMin. I would suggest if we do merge in this ~as is that you delete the commented out lines which are not comments since this is only adding noise. You could also consider pep8 variable/function names.